generators/filesystem/testing/expected.csv

```diff
@@ -0,0 +1,46 @@
+entity,attribute,value
+./foo.json,uid,1288452253

```

Comment:
Reminding you of [this comment](https://github.com/chop-dbhi/origins-generators/pull/6/files#r33710055).